clients/cli/src/workers/online.rs

```diff
@@ -479,7 +476,7 @@ async fn fetch_new_tasks_batch(
             Err(OrchestratorError::Http { status: 429, .. }) => {
                 let _ = event_sender
                     .send(Event::task_fetcher_with_level(
-                        "⏳ Rate limited during batch fetch".to_string(),
+                        "\t\tEvery node in the Prover Network is rate limited to 3 tasks per 3 minutes".to_string(),
```

Comment:
At some point it would be great to get this information from the server so that it can change dynamically with rate limiting.